mapslang/Makefile

```diff
@@ -74,13 +74,13 @@ else
   DEBUGME = no
 endif
 # Names of utility commands
-SLANG_URL          = https://www.jedsoft.org/snapshots/slang-pre2.3.3-24.tar.gz
-SLANG_NAME         = slang-pre2.3.3-24
-SLANG_ARCHIVE      = ../lib/slang.tgz
+SLANG_URL          = https://www.jedsoft.org/releases/slang/slang-2.3.2.tar.bz2
```

Comment:
I can't compile `mapslang` in this PR.

```windres -i ../../../mapslang/libslang-patches/libslang_private.rc --input-format=rc -o gw32objs/libslang_private.res -O coff
rm -f gw32objs/libslang.a
gcc -shared -static-libgcc -o gw32objs/libslang.dll gw32objs/slang.o  gw32objs/slarray.o  gw32objs/slclass.o  gw32objs/slcmd.o  gw32objs/slerr.o  gw32objs/slgetkey.o  gw32objs/slkeymap.o  gw32objs/slmalloc.o  gw32objs/slmath.o  gw32objs/slarith.o  gw32objs/slassoc.o  gw32objs/slmemchr.o  gw32objs/slmemcmp.o  gw32objs/slmemcpy.o  gw32objs/slmemset.o  gw32objs/slmisc.o  gw32objs/slparse.o  gw32objs/slprepr.o  gw32objs/slregexp.o  gw32objs/slrline.o  gw32objs/slsearch.o  gw32objs/slsmg.o  gw32objs/slstd.o  gw32objs/sltoken.o  gw32objs/sltypes.o  gw32objs/slscroll.o  gw32objs/slsignal.o  gw32objs/slsig.o  gw32objs/slkeypad.o  gw32objs/slerrno.o  gw32objs/slstring.o  gw32objs/slstruct.o  gw32objs/slcmplex.o  gw32objs/slarrfun.o  gw32objs/slimport.o  gw32objs/slpath.o  gw32objs/slcompat.o  gw32objs/slposdir.o  gw32objs/slstdio.o  gw32objs/slproc.o  gw32objs/sltime.o  gw32objs/slstrops.o  gw32objs/slbstr.o  gw32objs/slpack.o  gw32objs/slintall.o  gw32objs/slistruc.o  gw32objs/slposio.o  gw32objs/slnspace.o  gw32objs/slarrmis.o  gw32objs/slospath.o  gw32objs/slscanf.o  gw32objs/slvideo.o  gw32objs/slw32tty.o  gw32objs/sllower.o  gw32objs/slupper.o  gw32objs/slischar.o  gw32objs/slutf8.o  gw32objs/slwcwidth.o  gw32objs/slwclut.o  gw32objs/slcommon.o  gw32objs/sllist.o  gw32objs/slexcept.o  gw32objs/slfpu.o  gw32objs/slboseos.o  gw32objs/slxstrng.o  gw32objs/libslang_private.res -Wl,--out-implib,gw32objs/libslang.a
gw32objs/slang.o: file not recognized: file format not recognized
collect2.exe: error: ld returned 1 exit status
make[2]: *** [Makefile:115: gw32objs/libslang.a] Error 1
make[2]: Leaving directory '/d/Downloads/ADiKtEd-edit_tileset/lib/slang-2.3.2/src'
make[1]: *** [Makefile:2: all] Error 2
make[1]: Leaving directory '/d/Downloads/ADiKtEd-edit_tileset/lib/slang-2.3.2'
make: *** [Makefile:167: libslang.dll] Error 2```

Reply:
looks like some error with cleanup or patch. Why `gcc` and not `i686-w64-mingw32-gcc` is used?

Reply:
Not sure what happened, took another crack at compiling it and now it works fine so I probably did something wrong